tests/rptest/tests/node_id_assignment_test.py

```diff
@@ -119,6 +122,33 @@ def test_rejoin_with_wrong_node_id(self):
                                  expect_fail=True)
 
 
+class NodeIdAssignmentParallel(RedpandaTest):
+    """
+    Test that adds several nodes at once, assigning each a unique node ID.
+    """
+    def __init__(self, test_context):
+        super(NodeIdAssignmentParallel,
+              self).__init__(test_context=test_context, num_brokers=6)
+
+    def setUp(self):
+        self.redpanda.start(self.redpanda.nodes[0:3],
+                            auto_assign_node_id=True,
+                            omit_seeds_on_idx_one=False)
+
+    @cluster(num_nodes=6)
+    def test_assign_multiple_nodes(self):
+        self.redpanda.start(self.redpanda.nodes[3:],
+                            auto_assign_node_id=True,
+                            omit_seeds_on_idx_one=False)
+
+        def check_num_nodes():
+            brokers = self.redpanda._admin.get_brokers()
+            return 6 == len(brokers)
```

Comment:
```suggestion
            return len(self.redpanda.nodes) == len(brokers)
```
because all the available nodes are used (`self.redpanda.nodes[3:],` in `start()`)

Reply:
Done